src/main/java/com/github/seratch/jslack/api/model/File.java

```diff
@@ -55,16 +56,29 @@
     @SerializedName("thumb_160")
     private String thumb160;
 
+    private Integer imageExifRotation;
+
+    @SerializedName("original_w")
+    private String originalWidth;
+    @SerializedName("original_h")
+    private String originalHeight;
+
+    private String deanimateGif;
+    private String pjpeg;
+
     private String permalink;
     private String permalinkPublic;
 
     private String editLink;
 
+    private boolean hasRichPreview;
+    @SerializedName("preview_is_truncated")
+    private boolean previewTruncated;
```

Comment:
This attribute seems to be absent [here](https://api.slack.com/types/file). But it surely works for long text attachments.